### PR TITLE
Fix translations collision on contact

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR is workaround for a translation collision on "contact":
- the action "contact" translates to "Contacter"
- the contact section in this theme footer translates to "Contact".

As Jinja does not support `pgettext` (see pallets/jinja#441), there is no proper way to differenciate the 2 string.
So this workaround switch the footer section name to the plural form "Contacts" (it is a list of contact point)

### CODE REVIEW

**Comment by vinyll** [comment]
👍

File: udata_gouvfr/theme/templates/footer.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
             </section>
 
             <section class="col-xs-6 col-sm-3 col-md-4 col-lg-4 social">
-                <h5>{{ _('Contact') }}</h5>
+                <h5>{{ _('Contacts') }}</h5>
```
Comment (oncletom) [inline udata_gouvfr/theme/templates/footer.html]: `Contact Us` can be another alternative.

Comment (noirbizarre) [inline udata_gouvfr/theme/templates/footer.html]: You're right.
You can submit another PR to do so :+1:
